internal/services/appservice/windows_web_app_resource_test.go

```diff
@@ -1087,6 +1087,15 @@ func TestAccWindowsWebApp_withNode14(t *testing.T) {
 			),
 		},
 		data.ImportStep("site_credential.0.password"),
+		{
+			Config: r.nodeWithAppSettings(data, "~14"),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("site_config.0.application_stack.0.node_version").HasValue("~14"),
+			),
+			ExpectNonEmptyPlan: true, // should only indicate that the app settings have changed
```

Comment:
This means that we're expecting a diff to happen which is what these changes are trying to fix/avoid so this needs to be removed
```suggestion
```

Reply:
Unless a `ForceNew` property of the resource in the test config was changed then adding an additional test step like you have done here will call the update method of the resource. With the changes made in this PR,  after calling the update we would be expecting no diff to exist i.e. an empty plan.

Reply:
Thanks, with your explanation and what I read in the code (terraform-plugin-testing) to run the tests, I understand better how the tests are run. So I made the requested change 👍